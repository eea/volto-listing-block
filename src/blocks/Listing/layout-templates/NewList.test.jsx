import NewList from './NewsList';

describe('Test New List', () => {
  it('Given an object/object, p.restapi title/token', () => {
    expect(<NewList items={[]} />).toMatchSnapshot();
  });
});
