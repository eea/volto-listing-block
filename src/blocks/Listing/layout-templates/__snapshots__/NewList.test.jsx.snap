// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test New List Given an object/object, p.restapi title/token 1`] = `
<CustomNewsListTemplate
  items={Array []}
/>
`;
